src/distro/distro.py

```diff
@@ -1078,6 +1082,47 @@ def uname_attr(self, attribute: str) -> str:
         """
         return self._uname_info.get(attribute, "")
 
+    def __resolve_chroot_symlink(self, link_location: str) -> str:
+        # only resolve symlink if root_dir is set and link is under it
+        if (
+            self.root_dir is None
+            or os.path.commonprefix([self.root_dir, link_location]) != self.root_dir
+        ):
+            return link_location
+
+        seen_paths = []
+        while True:
+            try:
+                resolved = os.readlink(link_location)
+            except OSError:  # includes case "not a symlink"
+                return link_location
+
+            if os.path.isabs(resolved):
+                # i.e. absolute path (regarding to the chroot), that we need to
+                # "move" back inside the chroot
+                resolved = os.path.splitdrive(resolved)[1].lstrip(os.sep)
+                if os.altsep is not None:
+                    resolved.lstrip(os.altsep)
```

Comment:
This seems to have Windows in mind but then drops the drive part altogether. Also I don't fully understand the idea behind using `os.altsep`. I think this needs at least another comment, and potentially also a bug fix.

Reply:
Yes, the idea was to "universally" remove absolute "terms" from the path, and I assumed we don't (and can't) know how passed paths are expressed.
On Windows, we need to remove any drive part. Additionally, where `os.altsep` is defined ([as on NT](https://github.com/python/cpython/blob/d4e64cd4b0ea431d4e371f9b0a25f6b75a069dc1/Lib/ntpath.py#L16)) we should strip it as well.

Reply:
`/` on Windows is network shares I guess?

Maybe these low-level bits can be extracted to a new method and then the four lines here could become a high-level one-line `resolved = self.__abs_path_join(self.root_dir, resolved)` or similar.

Reply:
> `/` on Windows is network shares I guess?

Not necessarily, on PowerShell for instance, you can use `/` as "regular" path separator :shrug: 

> Maybe these low-level bits can be extracted to a new method and then the four lines here could become a high-level one-line `resolved = self.__abs_path_join(self.root_dir, resolved)` or similar.

Okay :pray: